src/views/MyDelegations.vue

```diff
@@ -100,7 +100,7 @@ export default class MyDelegations extends Vue {
           console.log(` No delegation`)
         } else {
           const candidateName = candidates[i].name == "" ? "Validator #" + (parseInt(i) + 1) : candidates[i].name
-          if(formatToCrypto(delegation.amount) > 0) {
```

Comment:
loom tokens have 18 decimal places so I believe you still have to format them in some way

Reply:
Sure, I'd recommend always passing around instances of BigNumber and then you can use the `isGreaterThan` method to do comparisons